Client/Frontend/Browser/Toast.swift

```diff
@@ -45,6 +45,10 @@ class Toast: UIView, ThemeApplicable {
         translatesAutoresizingMaskIntoConstraints = false
 
         DispatchQueue.main.asyncAfter(deadline: .now() + delay) {
+            if let currentToast = Toast.currentToast {
```

Comment:
I understand the reason for this change but I wonder if we should investigate instead why it was not necessary for the regular tabs undo toast.

Reply:
I investigated the problem, and without this change, the implementation only partially works. The undo toast does not stack only when we close the tabs from the X button because the handleTap method gets called on the view, dismissing the previous toast. However, when we close the tabs by swiping, the toasts stack one upon another.